modules/drivers/snowflake/test/metabase/driver/snowflake_test.clj

```diff
@@ -180,40 +180,42 @@
 (deftest week-start-test
   (mt/test-driver :snowflake
     (testing "The WEEK_START session setting is correctly incorporated"
-      (letfn [(invalidate-pool! []
-                (sql-jdbc.conn/invalidate-pool-for-db! (mt/db)))
```

Comment:
Manually invalidating the connection pool like we do in tests is no longer needed so I removed it here